deepinv/transform/scale.py

```diff
@@ -9,7 +9,7 @@ def sample_from(values, shape=(1,), dtype=torch.float32, device="cpu"):
     values = torch.tensor(values, device=device, dtype=dtype)
     N = torch.tensor(len(values), device=device, dtype=dtype)
     indices = torch.floor(N * torch.rand(shape, device=device, dtype=dtype)).to(
-        torch.int
+        torch.long
```

Comment:
I changed the type for the variables because torch gave me an error when running `pytest`. But I think that was because I used an older version of `torch` (I had also errors on other tests), when I performed the update, I was able to run the validation. I included this in the PR to be robust w.r.t. different versions of `torch`.

Reply:
I think it is fine